readthedocs/projects/forms.py

```diff
@@ -479,9 +479,8 @@ def __init__(self, *args, **kwargs):
                 self.fields.pop(field)
 
         # Remove analytics from new dashboard
-        if settings.RTD_EXT_THEME_ENABLED:
-            for field in ["analytics_code", "analytics_disabled"]:
-                self.fields.pop(field)
+        for field in ["analytics_code", "analytics_disabled"]:
+            self.fields.pop(field)
```

Comment:
Looks like we can just don't define these attributes since they are always removed.